### PR TITLE
Tickets/preops 4460

### PR DESCRIPTION
**Pull request by ChristinaAdair**
Update header for document-based tutorials to add RSP aspect

### CODE REVIEW

File: tutorials-examples/cmdline-beginner.rst [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 .. This section should provide a brief, top-level description of the page.
 
+**RSP Aspect:** Command-Line
```
Comment (MelissaGraham) [inline tutorials-examples/cmdline-beginner.rst:23]: Now that I see this I realize it should probably be the following, b/c the only aspects are Notebook, Portal, and API. And same for the other command-line tutorial.

**RSP Aspect:** Notebook (terminal command-line)

**Review by MelissaGraham — APPROVED** [review]
Once the updates to say " **RSP Aspect:** Notebook (terminal command-line) " are made, OK to merge (so 'approving' PR despite requesting changes b/c re-review is not needed).

File: tutorials-examples/cmdline-beginner.rst (hunk 1 above)
Comment (ChristinaAdair) [reply tutorials-examples/cmdline-beginner.rst:23]: Okay!  I'll change it

Comment (ChristinaAdair) [reply tutorials-examples/cmdline-beginner.rst]: changed

**Pull request merged**